packages/flutter/lib/src/widgets/interactive_viewer.dart

```diff
@@ -296,6 +300,16 @@ class InteractiveViewer extends StatefulWidget {
   /// than maxScale.
   final double minScale;
 
+  /// Changes the scrolling behavior after end of a gesture.
+  ///
+  /// Used as the coefficient of friction in the inertial translation animation.
+  /// This value was eyeballed to give a feel similar to Google Photos.
+  ///
+  /// Defaults to 0.0000135.
```

Comment:
What if we define a const variable locally in this file, named something along the lines "_kDefaultScrollFrictionCoefficient". That way, if the value is updated the documentation can not be out of sync.

```dart
/// This value was eyeballed to give a feel similar to Google Photos.
_kDefaultScrollFrictionCoefficient = 0.0000135;
```

```suggestion
  /// Defaults to [_kDefaultScrollFrictionCoefficient].
```